index.html

```diff
@@ -24,10 +24,11 @@
       <a href="#">Roshan Bajgain</a>
       <img class="menu" src="Union.png" alt="Menu">
       <div class="menu-items">
+        <img class="close-btn" src="x.svg" alt="close-botton">
         <ul class="nav1">
-          <li>portfolia</li>
-          <li>About</li>
-          <li>Contact</li>
+          <li><a class="information" href="#">Portfolio</a></li>
+          <li><a class="information" href="#about-myself">About</a></li>
+          <li><a class="information" href="#contact">Contact</a></li>
```

Comment:
* I suggest you stick to the design template as much as possible your mobile menu background colour, close icon(X) and text colour looks different kindly make the necessary adjustments 🙏 
![image](https://user-images.githubusercontent.com/52497246/132314477-25bcb778-1e59-44f0-8a26-0819b467f52d.png)
![image](https://user-images.githubusercontent.com/52497246/132314501-5544f0cf-e849-46f0-9b05-0d4732324bea.png)